core-bundle/src/Twig/Loader/TemplateLocator.php

```diff
@@ -101,7 +101,6 @@ public function findTemplates(string $path): array
             ->files()
             ->in($path)
             ->name('/(\.twig|\.html5)$/')
-            ->sortByName()
```

Comment:
If that is a bug in the `Finder` class, it should be fixed by Symfony, shouldn't it?

Reply:
IMHO this could probably improved in the `Finder` class but isn't a bug per se.